control/control_test.go

```diff
@@ -2,20 +2,30 @@ package control
 
 import (
 	"fmt"
-	"io/ioutil"
 	"math/rand"
 	"net"
-	"net/http"
 	"os"
 	"path/filepath"
-	"strings"
 	"testing"
 	"time"
 
+	"github.com/joyent/containerpilot/events"
 	"github.com/joyent/containerpilot/tests"
 	"github.com/joyent/containerpilot/tests/assert"
 )
 
+type TestApp struct {
```

Comment:
Aye, good point. I'll do this and expand test coverage as needed as I start to hit the rest of the endpoints.